### DISCUSSION TITLE
using qodana gradle plugin in a gradle multi project projects

### DISCUSSION THREAD

**Discussion by christian-draeger** [JetBrains/Qodana#65] [Q&A] [answered]
Hey,
what is the correct way to run inspections against all modules of a multi module project and is it possible to get an aggregated report?

**Comment by tiulpin** [comment, marked as answer]
Hey! 

Currently, I have a multi-module project where I set the plugin in the root `build.gradle.kts`
<img width="780" alt="Qodana Gradle plugin" src="https://user-images.githubusercontent.com/13538286/151521078-67ac5478-9bdb-41f8-9f0f-e542c5129d80.png">

It's also possible to specify the configuration there.

Regarding your question about the aggregated reports: you'll be able to see all of the issues in the report at once. Still, you'll also be able to navigate directories (modules).

<img width="524" alt="CleanShot 2022-01-28 at 12 27 06@2x" src="https://user-images.githubusercontent.com/13538286/151521657-abda7f86-d9ec-4892-afdd-d509df71e6a2.png">
